src/PerfView/EventViewer/EventWindow.xaml

```diff
@@ -242,6 +242,13 @@
                         MinWidth="60"
                         IsReadOnly="True" />
                         <WPFToolKit:DataGridTextColumn
+                        x:Name="TimeStampColumn"
+                        Header="TimeStamp"
+                        Binding="{Binding TimeStamp, StringFormat=u}"
```

Comment:
StringFormat u (Universal) is compact, but it lacks precision (nearest second).
Consider `StringFormat=o` (ISO8601) which has more precision.

Reply:
Fixed